.github/workflows/build.yaml

```diff
@@ -0,0 +1,199 @@
+        # If the file does not exist when tar excludes it, then it will not
+        # actually exclude it, so it must first be touched
+        run: |
+          touch ola-${{ matrix.id }}-source-tree.tar.gz
+          tar --exclude=ola-${{ matrix.id }}-source-tree.tar.gz -cvzf ola-${{ matrix.id }}-source-tree.tar.gz .
+      - name: SHA256 artifact archives
+        if: always()
+        run: sha256sum ola-*.tar.gz
+      - name: Upload source tree artifact
+        uses: actions/upload-artifact@v3
+        if: always()
+        with:
+          name: ola-${{ matrix.id }}-source-tree
+          path: ola-${{ matrix.id }}-source-tree.tar.gz
+      - name: Upload built artifact
+        if: matrix.task == 'distcheck' || matrix.task == 'dist'
+        uses: actions/upload-artifact@v3
+        with:
+          name: ola-${{ matrix.id }}-dist
+          path: |
+            ola-*.tar.gz
+            !ola-${{ matrix.id }}-source-tree.tar.gz
+      - name: Install coverage tools
+        if: matrix.task == 'coverage'
+        run: apt-get -y install curl
+      - name: Upload coverage to Coveralls
+        if: matrix.task == 'coverage'
+        uses: coverallsapp/github-action@v2
+        with:
+          # Coveralls GitHub action does not support its own format
+          # see: https://github.com/coverallsapp/github-action/issues/104
+          # file: coverage/coverage.coveralls.json
+          file: coverage/coverage.cobertura.xml
+          format: cobertura
+          flag-name: ${{ matrix.id }}
+      - name: Upload coverage artifacts
+        if: always() && matrix.task == 'coverage'
+        uses: actions/upload-artifact@v3
+        with:
+          name: ola-${{ matrix.id }}-coverage
+          path: coverage/
+  verify-trees:
+    name: 'Verify trees for ${{ matrix.id }}'
+    needs: build
+    if: "always()" # Run if some builds fail but ensure they all complete first
+    container: debian:stable
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - id: "distcheck-debian-stable-amd64-gcc"
+          - id: "distcheck-debian-stable-amd64-clang"
+    steps:
+      - name: Download built source tree archive
+        uses: actions/download-artifact@v3
+        with:
+          name: ola-${{ matrix.id }}-source-tree
+          path: .
+      - name: SHA256 artifact archive
+        run: sha256sum ola-${{ matrix.id }}-source-tree.tar.gz
+      - name: Unarchive artifacts and delete archive
+        shell: bash
+        run: |
+          tar -xvzf ola-${{ matrix.id }}-source-tree.tar.gz .
+          rm ola-${{ matrix.id }}-source-tree.tar.gz
+      - name: Display structure of extracted files
+        if: env.ACTIONS_STEP_DEBUG == 'true'
+        run: ls -alR
+      - name: Update package database
+        run: apt-get update -y
+      - name: Install Python
+        run: apt-get -y install python3 python-is-python3
+      # Disable problem matcher for now until verify trees is fixed
```

Comment:
```suggestion
      # TODO(Perry): Disable problem matcher for now until verify trees is fixed
```

Reply:
Committed!

Reply:
Thanks @DaAwesomeP . It wouldn't let me edit your code, maybe you've disabled contributor access?